### DISCUSSION TITLE
Print settings

### DISCUSSION THREAD

**Discussion by timmeade** [aderusha/DDD-Printable-Wall-Control-System#54] [Q&A]
This looks like an awesome project.   We are very excited to try it.

Is there any documentation on print settings?

I have tried to print a quick hook and the retainer three times.   .3 .2 and .1 layers.  I've not been able to get the retainer installed.   
I'm guessing it's the print setting etc.   Is there a definitive list of what they should be?

Thanks

Tim

**Comment by aderusha** [comment]
The vertical slots should be 2.2mm wide and that's what the tabs are set for.  0.2mm layer height should work if your printer is properly calibrated, specifically on the Z axis.

I have found that some WC panels, particularly those purchased from their scratch & dent outlet (cheappegboards.com), have a little bit extra powder coat, such that the slots are slightly less thick than 2.2mm.  A few strokes of a needle file is enough to open that back up to spec.

**Comment by leviable** [comment]
This is the first time I've really cared about the quality of my prints, and I was having considerable problems with "elephant foot". Elephant foot is when the bottom layer(s) mushroom out a little bit, due to a variety of potential factors. This affects printing of wall control components because the extra bit of mushrooming causes the 10mmx8mm pin holes to be to be too tight, and the embedded pins on the center pieces to be to wide, resulting in parts that simply cant fit together without filing the elephants foot and/or using an undo amount of force.

So I did a bunch of tweaking and settled on the following:

1. Bed levelling is important, but don't over fixate on it. I spent a ton of time leveling and releveling before realizing my issues with elephants foot and bed adhesion were also related to temperatures.[ I used this project to level my bed and do test prints](https://www.thingiverse.com/thing:2987803). I also found[ this youtube tutorial](https://youtu.be/_EfWVUJjBdA?t=215) helpful, particularly the part where he runs his fingers lightly over the test bead to see how well it is adhering to the surface. 
2. Do a series of test prints, varying **either** the bed temperature or the extruder temperature in 5 degree increments. I did a series of prints at 195/50, 200/50, 205/50, etc, and ultimately found that extruder temp of 215 and bed temperature of 60 worked well.
3. Investigate different extruder/bed temperatures for the initial layer, and different temperatures for the remainder of the print. I further found that initial temperatures of 215/45 gave me the best printing results for the first layer, eliminating elephants foot in its entirety.

These are my Cura settings I'm now using for printing to my Ender 3 Pro
![image](https://user-images.githubusercontent.com/11619428/216344497-f6f5c42f-525c-456a-959b-72d7053ff2d7.png)
![image](https://user-images.githubusercontent.com/11619428/216344590-8fa4747b-284c-4fd4-881a-f1e7c3cdad0e.png)
